acoular/fastFuncs.py

```diff
@@ -545,8 +545,10 @@ def _freqBeamformer_EigValProb_SpecificSteerVec_CsmRemovedDiag(
 
 # %% Point - Spread - Function
 def calcPointSpreadFunction(steerVecType, distGridToArrayCenter, distGridToAllMics, waveNumber, indSource, dtype):
-    r"""Calculates the Point-Spread-Functions. Use either a predefined steering vector
-    formulation (see :ref:`Sarradj, 2012<Sarradj2012>`) or pass it your own steering vector.
+    r"""Calculates the Point-Spread-Functions.
```

Comment:
```suggestion
    r"""
    Calculates the Point-Spread-Functions.
```

Again `GL01 Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)`